### PR TITLE
8271328: User is able to choose the color after disabling the color chooser.

### PR DESCRIPTION
**Pull request by kumarabhi006**
The proposed solution for this defect is to check the color chooser enabled state. If it is disabled then disable all the components (spinner, textfield etc.) in panel and don't process the mouse event when user click on a color triangle or color wheel.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8271328](https://bugs.openjdk.org/browse/JDK-8271328): User is able to choose the color after disabling the color chooser.

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**)
 * [Tejesh R](https://openjdk.org/census#tr) (@TejeshR13 - Author)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/10276/head:pull/10276` \
`$ git checkout pull/10276`

Update a local copy of the PR: \
`$ git checkout pull/10276` \
`$ git pull https://git.openjdk.org/jdk pull/10276/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 10276`

View PR using the GUI difftool: \
`$ git pr show -t 10276`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/10276.diff">https://git.openjdk.org/jdk/pull/10276.diff</a>

</details>

### CODE REVIEW

File: test/jdk/javax/swing/JColorChooser/TestDisabledColorChooser.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ * @run main TestDisabledColorChooser
+ */
+
+import java.awt.Color;
+import java.awt.event.InputEvent;
+import java.awt.event.ItemEvent;
+import java.awt.event.ItemListener;
+import java.awt.Point;
+import java.awt.Robot;
+import javax.swing.*;
```
Comment (TejeshR13) [inline test/jdk/javax/swing/JColorChooser/TestDisabledColorChooser.java:39]: Please import only used swing components/functionalities. Can remove the generic `swing.*` import.

Comment (kumarabhi006) [reply test/jdk/javax/swing/JColorChooser/TestDisabledColorChooser.java]: Updated the import statements.

**Review by prsadhuk — APPROVED** [review]

**Review by TejeshR13 — APPROVED** [review]

**Pull request closed without merging**